README.md

```diff
@@ -0,0 +1,42 @@
+# VulnTotal Browser Extension
+
+## Overview
+The VulnTotal Browser Extension integrates the VulnTotal vulnerability analysis tool directly into your browser. By leveraging Pyodide, the extension allows you to perform vulnerability checks on software packages without relying on server-side components. The extension provides a user-friendly interface that displays analysis results.
+
+## Features
+- Runs VulnTotal directly in the browser using Pyodide.
+- Displays vulnerability analysis results in a clear, table format.
+
+## Installation
+1. Clone the repository:
+``` sh
+git clone https://github.com/aboutcode-org/vulntotal-extension.git
+cd vulntotal-extension
+```
+2. Build the Extension:
+- On Linux, run:
+```sh
+./release.sh
+```
+- On Windows, run:
+```sh
+release.bat
+```
+This will generate the extension file (.crx) in the dist folder.
+
+## Load the Extension:
```

Comment:
Also add instruction to load extension in firefox.

Reply:
We can't install crx extensions manually on firefox, I believe we'll add a section later for download links when it gets published on extensions stores for chrome, firefox, edge..etc